mlx/c/fast.cpp

```diff
@@ -3,6 +3,8 @@
 /* This file is auto-generated. Do not edit manually. */
 /*                                                    */
 
+#include <cstdarg>
```

Comment:
This is an auto-generated file. I do not get the error there as well, so curious to know where it occurs. For this one, and the one in `metal.cpp`, I would suggest to put additional includes in `mlx/c/private/mlx.h`

Reply:
`cstdarg` was for the `va_x` functions, like `va_list`, `va_arg` etc.

Reply:
Noted, i.e. moving it to `mlx/c/private/mlx.h`.